docker-compose.yml

```
@@ -44,6 +44,20 @@ services:
     env_file:
       -  ./backend/.env
 
+  swagger:
+    image: swaggerapi/swagger-ui:latest
+    ports:
+      - 8001:8080
+    volumes:
+      - ./api.yml:/api.yml
```

ルート直下に置くのではなく、他のコンテナど同様にディレクトリを作るのはどうでしょう？

例えば
```suggestion
      - ./swagger/api.yml:/api.yml
```
など、いかがでしょうか。
（あるかわからないですが）ファイル追加もできるでしょうし、何よりこのコンテナ特有のファイルということが明確になると思いました。

---

了解です。修正します！